core/templates/pages/contributor-dashboard-admin-page/contributor-dashboard-admin-page.module.ts

```diff
@@ -70,62 +60,20 @@ import { AppErrorHandlerProvider } from 'pages/oppia-root/app-error-handler';
     ContributorAdminDashboardPageComponent,
     ContributorAdminStatsTable,
     TopicFilterComponent,
-    UsernameInputModal
+    UsernameInputModal,
+    ContributorDashboardAdminPageComponent,
+    ContributorDashboardAdminPageRootComponent
   ],
   entryComponents: [
     CdAdminTranslationRoleEditorModal,
     CdAdminQuestionRoleEditorModal,
     ContributorDashboardAdminNavbarComponent,
     ContributorAdminDashboardPageComponent,
+    ContributorDashboardAdminPageComponent,
```

Comment:
The functionalities are completely different in the two components but they are not independent of one another.

Reply:
Can you please check and help understand their usage? [I'm asking this as the name of these components are too similar and looks conflicting]

Reply:
@DubeySandeep I am really sorry sir. Both components are similar but with different UI. ```contributor-admin-dashboard``` is the component with new UI.

I think this new ```contributor-admin-dashboard``` UI is still in development stage. Probably because of this very reason the ```contributor-dashboard-admin page``` component was not migrated to angular.

Reply:
I see, thanks for the explanation!